SimPEG/EM/Static/DC/IODC.py

```diff
@@ -0,0 +1,383 @@
+import numpy as np
+import SimPEG
+from SimPEG import Utils, Mesh
+from . import SrcDC as Src   # Pole
+from . import RxDC as Rx
+from .SurveyDC import Survey_ky
+import matplotlib.pyplot as plt
+import matplotlib
+
+
+class IO(object):
+    """Input and Output for DC, IP, SP, ..."""
+
+    # TODO: use properties
+
+    Alocs = None
```

Comment:
Yep, I think this will help a bunch of things. as well as improve the documentation. I also like the idea of introducing new names with pep8 compatibility (lowercase, underscores and longer/explicit names).

Reply:
sounds great to me.